README.md

```diff
@@ -1,23 +1,24 @@
+<div align="center">
+
 # EQ Author
 
+<img width="180" height="180" src="eq-author/public/android-chrome-256x256.png" alt="eq-author logo">
+
 [![Build Status](https://travis-ci.com/ONSdigital/eq-author-app.svg?branch=master)](https://travis-ci.com/ONSdigital/eq-author-app)
 [![codecov](https://codecov.io/gh/ONSdigital/eq-author-app/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-author-app)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/f9cbc9e5c531436ca73ee0769a766678)](https://www.codacy.com/app/jchapman68/eq-author-app?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/eq-author-app&amp;utm_campaign=Badge_Grade)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/f9cbc9e5c531436ca73ee0769a766678)](https://www.codacy.com/app/jchapman68/eq-author-app?utm_source=github.com&utm_medium=referral&utm_content=ONSdigital/eq-author-app&utm_campaign=Badge_Grade)
```

Comment:
This contains a username in - is that necessary?

Reply:
I think it is, I could change with with `[![Codacy Badge](https://api.codacy.com/project/badge/Grade/87418ffa08ad4931aa7e08946f7b2d86)](https://www.codacy.com/app/heofs/eq-author-app?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/eq-author-app&amp;utm_campaign=Badge_Grade)`, but it will just contain my username then...

Reply:
Would be good to link to a generic ONS user if possible so it doesn't break when user leaves ONS.

Reply:
Hmm.. which user would that be?

Reply:
Seems `ONSDigital` works as a user!